### PR TITLE
deprecate [redmine] plugin rating badges

### PR DESCRIPTION
**Pull request by chris48s**
Redmine badges have been failing for some time due to a consistent 500 error from the upstream API.

- https://img.shields.io/redmine/plugin/stars/redmine_xlsx_format_issue_exporter
- https://www.redmine.org/plugins/redmine_xlsx_format_issue_exporter
- https://www.redmine.org/plugins/redmine_xlsx_format_issue_exporter.xml

The badge receives little traffic and despite being broken for months, nobody has complained. If we a look at a bunch of different redmine plugins, we see the same thing for all of them:

- https://www.redmine.org/plugins/redmine_issue_dynamic_edit.xml
- https://www.redmine.org/plugins/redmine-x-upgrade-plugin.xml
- https://www.redmine.org/plugins/redmine-x-gantt-chart-plugin.xml
- https://www.redmine.org/plugins/redmine_issue_to_email.xml
- etc

Suggest we deprecate this service.

### CODE REVIEW

File: services/redmine/redmine.service.js [hunk 1]
```diff
@@ -1,79 +1,21 @@
...
-      color: floorCountColor(rating, 2, 3, 4),
-    }
-  }
-}
-
-export { RedminePluginRating, RedminePluginStars }
+    pattern: ':various*',
+  },
+  label: 'redmine',
+  dateAdded: new Date('2023-09-11'),
```
Comment (calebcartwright) [inline services/redmine/redmine.service.js:20]:
```suggestion
  dateAdded: new Date('2023-09-14'),
```

File: services/redmine/redmine.service.js [hunk 2]
```diff
@@ -1,79 +1,21 @@
...
-      label: 'rating',
-      message: `${rating.toFixed(1)}/5.0`,
-      color: floorCountColor(rating, 2, 3, 4),
-    }
-  }
-}
+    pattern: ':various*',
+  },
+  label: 'redmine',
+  dateAdded: new Date('2023-09-11'),
```
Comment (calebcartwright) [inline services/redmine/redmine.service.js:10]:
```suggestion
  dateAdded: new Date('2023-09-14'),
```

**Review by calebcartwright — COMMENTED** [review]
sounds reasonable, and changes look good. going to bump the date a tad and then merge

**Comment by chris48s** [comment]
The reason this didn't auto-merge is because it was missing an approving review.

**Pull request merged**